go.mod

```diff
@@ -5,34 +5,51 @@ go 1.17
 require (
 	github.com/DATA-DOG/go-sqlmock v1.5.0
 	github.com/go-sql-driver/mysql v1.6.0
-	github.com/google/go-cmp v0.5.6 // indirect
 	github.com/hashicorp/hcl/v2 v2.10.0
 	github.com/lib/pq v1.10.4
 	github.com/manifoldco/promptui v0.9.0
 	github.com/spf13/cobra v1.3.0
 	github.com/stretchr/testify v1.7.1-0.20210427113832-6241f9ab9942
 	github.com/zclconf/go-cty v1.8.0
 	golang.org/x/mod v0.5.1
-	golang.org/x/sys v0.0.0-20211205182925-97ca703d548d // indirect
-	golang.org/x/text v0.3.7 // indirect
 )
 
 require (
+	github.com/docker/docker v20.10.13+incompatible
+	github.com/docker/go-connections v0.4.0
```

Comment:
Wonder if it's not simpler to just use os.Exec (depends on the lib size).

Reply:
I cannot reliably shut down / destroy containers and have to do string parsing to get things like ID etc. 

Which eventually will make me rewriting the lib...